src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAroundBasic.java

```diff
@@ -235,6 +235,12 @@ void forIterator()
         int i = (int) ( 2 / 3 );
         return null;
     }
+
+    void forColon() {
+        int ll[] = new int[10];
+        for (int x:ll) {} // warn
```

Comment:
please add cases like
` for(int k: array)` and `for(int t :array)`

Reply:
done